workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-token-definitions/client-extension.yaml

```diff
@@ -0,0 +1,9 @@
+assemble:
+    - from: build/buildTheme/img
+      into: static/img
+liferay-sample-theme-token-definitions:
```

Comment:
Can you rename to liferay-sample-theme-frontend-token-definition. Same for the folder.